cli/launchpad/draw.go

```diff
@@ -0,0 +1,141 @@
+package launchpad
+
+import (
+	"fmt"
+	"io"
+	"strings"
+	"os"
+)
+
+// diagram represents a GCP draw diagram
+type diagram struct {
+	paths []*diagramPath
+	group *diagramGroup
```

Comment:
We should map this to the diagram syntax. 

Group essentially is a wrapper for a list of card and other nested groups. And as this code indicates, top level elements{} essentially equates to a group. Therefore, maybe a new name to avoid conflict 
Maybe something similar to this? 
```
type diagram struct {
  meta *diagramMeta
  elements *diagramElements
  paths []*diagramPath
}
type diagramElements diagramGroup
```